sound/soc/codecs/rt700-sdw.c

```diff
@@ -228,6 +225,9 @@ static int rt700_sdw_probe(struct sdw_slave *slave,
 	if (slave->status == SDW_SLAVE_ATTACHED)
 		rt700_io_init(&slave->dev, slave);
 
+	/* Assign ops */
+	slave->ops = &rt700_slave_ops;
+
```

Comment:
sorry @bardliao that doesn't look good. We shouldn't have any interrupt callback invoked before the update_status, which will do the inits.

Reply:
@plbossart Ok, let's work on making sure interrupt callback only be invoked after update_status.